src/scheduler/LogicCompiler.jl

```diff
@@ -70,7 +70,12 @@ end
 
 function compile_logic_constant(node)
     if node.kind === immediate
-        node.val
+        val = node.val
+        if Base.isoperator(Symbol(val))
+            Symbol(val)
+        else
+            val
+        end
```

Comment:
I think we could just always return `Symbol(val)` any way - I see that it's no-op for functions:
```julia
julia> :(1 << $(Symbol(map)) >>= 1)
:(1 << map >>= 1)
```

Reply:
I'm not sure this is always true. Different functions in different modules can have the same name.